usersService/src/main/java/il/ac/afeka/usersservice/boundaries/DepartmentBoundary.java

```diff
@@ -0,0 +1,62 @@
+package il.ac.afeka.usersservice.boundaries;
+
+import il.ac.afeka.usersservice.data.DepartmentEntity;
+
+public class DepartmentBoundary {
+
+    private String deptId;
+    private String departmentName;
+    private String creationDate;
+
+    public DepartmentBoundary() {
+    }
+
+    public DepartmentBoundary(DepartmentEntity dep) {
+        this.deptId = dep.getDeptId();
+        this.departmentName = dep.getDepartmentName();
+        this.creationDate = dep.getCreationDate();
+    }
+
+    public String getDeptId() {
+        return deptId;
+    }
+
+    public String getDepartmentName() {
+        return departmentName;
+    }
+
+    public String getCreationDate() {
+        return creationDate;
+    }
+
+    public void setDeptId(String deptId) {
+        this.deptId = deptId;
+    }
+
+    public void setDepartmentName(String departmentName) {
+        this.departmentName = departmentName;
+    }
+
+    public void setCreationDate(String creationDate) {
+        this.creationDate = creationDate;
+    }
+
+    public DepartmentEntity toEntity() {
+        DepartmentEntity entity = new DepartmentEntity();
+        entity.setDeptId(this.getDeptId());
+        entity.setDepartmentName(this.getDepartmentName());
+        entity.setCreationDate(this.getCreationDate());
```

Comment:
```suggestion
        entity.setCreationDate(LocalDate.parse("dd-MM-yyyy").toString());
```
Creation date should be determined by the system and not by the user.